src/utilities/process-readme.js

```diff
@@ -8,6 +10,16 @@ module.exports = function processREADME(body) {
     .replace(/<h1.*?>.+?<\/h1>/, '')
     .replace(/# .+/, '')
     .replace(/.*\n=+/, '')
+    // Replace local github links with absolute links to the github location
```

Comment:
This change broke some readmes:
From: https://raw.githubusercontent.com/webpack-contrib/polymer-webpack-loader/master/README.md
Original links:
```md
[![npm version](https://badge.fury.io/js/polymer-webpack-loader.svg)](https://badge.fury.io/js/polymer-webpack-loader)
[![build status](https://travis-ci.org/webpack-contrib/polymer-webpack-loader.svg?branch=master)](https://travis-ci.org/webpack-contrib/polymer-webpack-loader)

> [Polymer](https://www.polymer-project.org/) component loader for [webpack](https://webpack.js.org/).
```

After processed by this line:
```md
undefined](https://badge.fury.io/js/polymer-webpack-loader)
undefined](https://travis-ci.org/webpack-contrib/polymer-webpack-loader)

> undefined component loader for undefined.
```

@Munter Could you please take a look, we could maybe avoid that transformation at all

Reply:
Fix: https://github.com/webpack/webpack.js.org/pull/2451/